pkg/controller/csr_check_test.go

```
@@ -78,6 +79,7 @@ func init() {
 	now = testingclock.NewFakePassiveClock(baseTime).Now
 	networkv1.AddToScheme(scheme.Scheme)
 	configv1.AddToScheme(scheme.Scheme)
+	egressipv1.AddToScheme(scheme.Scheme)
```

golangci-lint (2.12.2)

[error] Error return value is not checked

(errcheck)